### ISSUE TITLE
[Bug] Input string was not in a correct format. (Number.Parsing.cs, 1781)

### ISSUE THREAD

**Issue by jkittrell** [DanWBR/dwsim6#705]
**Describe the bug**
Created compound with compound creator ("FDCA") which is a solid, unifac group assignments used to estimate compound properties, saved as JSON file.  Imported into simulation, simple stream mixing.  NRTL property pkg selected, modified to SVLE phase equilibria and check solids handling to force this compound into solids form.  Getting an error on the material stream:

"Errors occurred during the calculation of the flowsheet, please read the following messages for more details.
[12/30/2024 3:17:25 PM] Error in 'FDCA 100%': Input string was not in a correct format. (Number.Parsing.cs, 1781)"

a. if i revert back to VLE/no solids phase equilibria, i get the same input string error
b. However, under VLE phase equilibria, if i change composition of this stream slightly to 99% FDCA/1% water, simulation solves, of course showing FDCA in liquid phase.
 
suggesting the JSON file may not be complete from compound creator, unifac estimates incorrect, or the cross-platform interface is having trouble with solids phase equilibrium.

**To Reproduce**
Attaching the json file from compound creator and screen print of results errors from within DWSIM results window if helpful
[2,5 FDCA (unifac).json](https://github.com/user-attachments/files/18277918/2.5.FDCA.unifac.json)
![Screenshot 2024-12-30 at 3 51 10 PM](https://github.com/user-attachments/assets/d45929bf-7e15-4265-9693-f971bc3290e5)

**Expected behavior**
selecting SVLE/Nested Loop should drive the specified compound into solid phase regardless of thermo pkg selected.

**Desktop (please complete the following information):**
 - OS: macOS 13.6.7
 - DWSim 8.8.3

**Additional context**

**Comment by DanWBR** [comment]
@jkittrell could you please attach the relevant files:

**Comment by jkittrell** [comment]
absolutely, thanks for taking a look
[FDCA Test.zip](https://github.com/user-attachments/files/18737563/FDCA.Test.zip)

**Comment by DanWBR** [comment]
@jkittrell you did this somehow:

![Image](https://github.com/user-attachments/assets/59e44567-f370-451a-8cef-b25c33cf48cf)

A blank value for SolidDensityEquation field should fix it. 

[2.5.FDCA 2.unifac.json](https://github.com/user-attachments/files/18739078/2.5.FDCA.2.unifac.json)

**Comment by jkittrell** [comment]
Yes, blank eliminated the issue, thank you so very much.  I'd like to define the density.  I see the note now to use '.' to separate numbers instead of comma.  I tried that syntax too and also resulted in incorrect format error.  Is my syntax incorrect perhaps?

**Comment by DanWBR** [comment]
@jkittrell probably
